pipelines/customer_orders_demo/code/customer_orders_demo/graph/Sum_Amounts.py

```
@@ -12,7 +12,7 @@ def Sum_Amounts(spark: SparkSession, in0: DataFrame) -> DataFrame:
     return df1.agg(
         count(col("order_id")).alias("orders"), 
         sum(col("amount")).alias("amounts"), 
-        (first(col("account_length_days")) + lit(101)).alias("account_length_days"), 
+        (first(col("account_length_days")) + lit(100)).alias("account_length_days"), 
```

Ruff (0.8.0)

`first` may be undefined, or defined from star imports

(F405)